### ISSUE TITLE
yavli update

### ISSUE THREAD

**Issue by smed79** [easylist/easylist#3109]
dead, parked or with no content sites
```
alfonzorachel.com
americanewshub.com
creepybasement.com
damnlol.com
dccrimestories.com
freedomdaily.com
girlsjustwannahaveguns.com
groopspeak.com
ifyouonlynews.com
instigatornews.com
janmorganmedia.com
libertyalliance.com
patriotnewswire.com
redhotchacha.com
silenceisconsent.net
theviralmob.com
usherald.com
winningdemocrats.com
xtribune.com
```

new yavli sites

```
appleeats.com
boomsbeat.com
breakingfirst.com
bugout.news
clashamerica.com
clashdaily.com
designbump.com
destinationtips.com
firstinfreedomdaily.com
glitch.news
greatamericanrepublic.com
hispolitica.com
itechpost.com
pjmedia.com
profitconfidential.com
sciencetimes.com
sportsworldnews.com
stupid.news
sweepstakesfanatics.com
thedesigninspiration.com
wakingtimes.com
```

Typo

`|healternativedaily.com|` should be `|thealternativedaily.com|`

**Comment by Khrin** [comment]
Thank you for your work.

I checked also the domains you've suggested to be added in the list, but not always I was able to find the unblocked adverts.
